eng/.docsettings.yml

```diff
@@ -9,8 +9,14 @@ omitted_paths:
   - samples/*
   - common/SmokeTests/*
   - sdk/*/samples/*
+  - sdk/*/*/samples/*
```

Comment:
What places do we have this structure for samples?

Reply:
Honestly, I'm not certain what the one ABOVE it is for. 

The one that's there currently matches something like:
```
sdk/eventhubs/samples
sdk/    *    /samples
```
Mine will match 
```

sdk/eventhubs/Azure.Messaging.Eventhubs/samples
sdk/    *    /             *           /samples
```

Which I feel like omits a bunch more samples than the one that they have.